src/test/java/com/karankumar/bookproject/backend/entity/AuthorTest.java

```diff
@@ -35,36 +35,36 @@
 
 @IntegrationTest
 class AuthorTest {
-    private static BookService bookService;
+    private final BookService bookService;
 
     private static Book testBook1;
     private static Book testBook2;
-    private static AuthorService authorService;
+    private final AuthorService authorService;
     private static PredefinedShelf toRead;
 
-    @BeforeAll
-    public static void setup(@Autowired PredefinedShelfService predefinedShelfService,
-                             @Autowired BookService bookService,
-                             @Autowired AuthorService authorService) {
+    public AuthorTest(final @Autowired PredefinedShelfService predefinedShelfService,
+                      final @Autowired BookService bookService,
+                      final @Autowired AuthorService authorService) {
```

Comment:
Could you please remove the `final` modifiers and the `@Autowired` annotations on each of the parameters? The constructor itself can be marked as `@Autowired`

Reply:
All changes done and pushed! You can check them now